content/tracing/languages/go.md

```diff
@@ -89,19 +89,46 @@ The Go tracer includes support for the following data stores and libraries.
 
 **Note**: The [integrations documentation][53] provides a detailed overview of the supported packages and their APIs, along with usage examples.
 
+Packages must be imported, i.e.:
+
+```go
+import "gopkg.in/DataDog/dd-trace-go.v1/contrib/<package_dir>/<package_name>"
+```
+
 ## Configuration
 
-The tracer is configured with options parameters when the `Start` function is called. A list of available options are:
+The tracer is configured with options parameters when the `Start` function is called. An example for generating a trace:
```

Comment:
👍 but I would mention the library used in this case: `HTTP` for context, such as _An example for generating a trace using the HTTP library_.